flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "end-of-life": "The application has been renamed to io.gitlab.liferooter.TextPieces.gnome.",
+  "end-of-life-rebase": "io.gitlab.liferooter.TextPieces.gnome"
```

Comment:
Wrong app id.

Reply:
Ah, thank you, missed it when changed the app ID